client/Components/ReviewFormModal.jsx

```diff
@@ -91,14 +95,25 @@ export default class ReviewFormModal extends React.Component {
     });
   }
 
+  sendReviewToProxy(review) {
```

Comment:
You might want to update the name of this function, these events are being sent to the window/browser, not the proxy server.